### PR TITLE
[jsk_spotkinova_robot] Update README.md

### PR DESCRIPTION
**Pull request by a-ichikura**
Add
・How to dock / undock spot
・When Ros cannot connect to Belka
・How to change pose with euslisp

@708yamaguchi

### CODE REVIEW

**Review by sktometometo — APPROVED** [review]

File: jsk_spotkinova_robot/README.md [hunk 1]
```diff
@@ -229,3 +230,33 @@ You can use `inverse-kinematics` to move arm.
...
+
+[Coral USB](https://github.com/knorth55/coral_usb_ros.git) is plugged into the Spotkinova's internel PC, so you can use it with spotkinova's camera.
+
+*This process needs ssh*
+
+```bash
+ssh ichikura@belka.local
+[password]
+source /opt/ros/melodic/setup.bash
+source /home/spot/spot_coral_ws/devel/setup.bash
```
Comment (708yamaguchi) [inline jsk_spotkinova_robot/README.md]:
```
ssh spot@belka.local
[password]
source /opt/ros/melodic/setup.bash
source $HOME/spot_coral_ws/devel/setup.bash
```
?

Comment (a-ichikura) [reply jsk_spotkinova_robot/README.md]: Yes, it is spot@belka.local.

File: jsk_spotkinova_robot/README.md [hunk 2]
```diff
@@ -97,26 +158,33 @@ If you want to send an angle-vector to kinova, you need to send the joint angles
...
+
+To change the full body pose, You can use `body-pose` method.
 ```
-(send *spotkinova* :head :move-end-rot -90 :z)
+(send *ri* :body-pose '(0 0.2 0))
+(send *ri* :body-pose '(0 -0.2 0))
+(send *ri* :body-pose '(0 0 0.2))
+(send *ri* :body-pose '(0 0 -0.2))
+(send *ri* :body-pose '(0.1 0 0))
+(send *ri* :body-pose '(-0.1 0 0))
```
Comment (708yamaguchi) [inline jsk_spotkinova_robot/README.md:172]: I am sorry, but could you please add information about coords?

Like

To change the full body pose, you can use `body-pose` method with rpy list or coords variable.
```
(send *ri* :body-pose '(0 0.2 0))
(send *ri* :body-pose '(0 -0.2 0))
(send *ri* :body-pose '(0 0 0.2))
(send *ri* :body-pose '(0 0 -0.2))
(send *ri* :body-pose '(0.1 0 0))
(send *ri* :body-pose '(-0.1 0 0))
(send *ri* :body-pose (make-coords :pos #f(0 0 50) :rpy #f(0.1 0 0)))
```

Comment (a-ichikura) [reply jsk_spotkinova_robot/README.md:172]: Ok, I will add it.

**Review by 708yamaguchi — APPROVED** [review]
Thank you very much for updating minor changes.

**Comment by mqcmd196** [comment]
+1

**Pull request merged**